instagram/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar.js';
+import PostContainer from './components/PostContainer/PostContainer.js';
+import PostsPage from './components/PostContainer/PostsPage.js';
+import Login from './components/Login/Login.js';
+import Authenticate from './components/Authentication/Authenticate.js';
+
+
+class App extends Component {
+  constructor() {
```

Comment:
You can remove this constructor and the contents inside of it. The constructor for a React component is called before it is mounted. Since this is not called before it is mounted, there is no need for it.